### DISCUSSION TITLE
I try to show image to table colums but error. I need your help

### DISCUSSION THREAD

**Discussion by tqt97** [awcodes/filament-curator#147] [Q&A]
when i try to use CuratorColumn to show image in List page of Filamentphp and i get error

> Awcodes\Curator\Components\Tables\CuratorColumn::getMedia(): Return value must be of type ?Awcodes\Curator\Models\Media, int returned

Help me, pls

**Comment by awcodes** [comment]
It needs to be a relationship to work in tables.
